### PR TITLE
Add Ore Devs

### PR DESCRIPTION
**Pull request by Inscrutable**

### CODE REVIEW

**Comment by Spongy** [comment]
A preview for this pull request is available at https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/c6cb058/.

Here are some links to the pages that were modified:

- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/c6cb058/about/staff.html

_Since the preview frequently changes, please link to [this comment](https://github.com/SpongePowered/SpongeDocs/pull/672#issuecomment-373663180), not to the direct url to the preview._

**Review by ST-DDT — APPROVED** [review]
Formatting approved.

File: source/about/staff.rst [hunk 1]
```diff
@@ -114,8 +114,24 @@ The **Systems Maintainers** ensure the efficient performance of Sponge's servers
...
 |                                   |                                    |
 |                                   | Web Development                    |
+|                                   |                                    |
+|                                   | Ore Developer                      |
 +-----------------------------------+------------------------------------+
 
+Ore Developers
+~~~~~~~~~~~~~~
+
+The **Ore Developers** work on producing, enhancing and fixing the Ore plugin distribution platform.
```
Comment (phase) [inline source/about/staff.rst]: Oxford comma? `producing, enhancing, and fixing`

File: source/about/staff.rst [hunk 2]
```diff
@@ -114,8 +114,24 @@ The **Systems Maintainers** ensure the efficient performance of Sponge's servers
 | progwml6                          | Systems Management                 |
 |                                   |                                    |
 |                                   | Web Development                    |
+|                                   |                                    |
+|                                   | Ore Developer                      |
```
Comment (phase) [inline source/about/staff.rst]: `Ore Management`?

**Pull request merged**